lib/phlex/translation.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Phlex
+	module Translation
+		def self.included(view)
+			view.extend(ClassMethods)
+		end
+
+		module ClassMethods
+			attr_writer :translation_path
```

Comment:
I moved the `virtual_path` logic to translation, and renamed it to `translation_path`, but if you think this information will be useful to other features, I can extract it to another module and rename back to `virtual_path`.
I decided to go with the memoization instead `included`/`inherited`, because the translation_path evaluation code will run only if `translate` or `t` methods are called, this can help the framework not degrade the performance.